pkg/scheduler/redis_scheduler.go

```diff
@@ -54,39 +60,103 @@ type RedisScheduler struct {
 // other cozy-stack processes to schedule jobs.
 func NewRedisScheduler(client *redis.Client) *RedisScheduler {
 	return &RedisScheduler{
-		client:  client,
-		stopped: make(chan struct{}),
+		client: client,
 	}
 }
 
 func redisKey(infos *TriggerInfos) string {
 	return infos.Domain + "/" + infos.TID
 }
 
+func eventsKey(domain string) string {
+	return "events-" + domain
+}
+
 // Start a goroutine that will fetch triggers in redis to schedule their jobs
 func (s *RedisScheduler) Start(b jobs.Broker) error {
 	s.broker = b
+	s.stopped = make(chan struct{})
+	go s.eventDispatcher()
```

Comment:
You're right, no need to spawn a goroutine here. fixed b4dec7b720b114949a7daa2b31823ea84d5f8f57